crates/core_simd/src/ord.rs

```diff
@@ -0,0 +1,222 @@
+use crate::simd::{intrinsics, LaneCount, Mask, Simd, SimdElement, SupportedLaneCount};
+
+/// Parallel `PartialOrd`.
+pub trait SimdPartialOrd {
```

Comment:
`SimdPartialOrd` should probably imply `SimdPartialEq`.

Reply:
Good idea, this also lets me remove the duplicate `Mask`